nrm_django/nrm_site/templates/admin/header.html

```diff
@@ -16,6 +16,9 @@
                 <li class="usa-nav__primary-item usa-nav__text">{% translate 'Welcome,' %}
                     <strong>{% firstof user.get_short_name user.get_username %}</strong>.
                 </li>
+                <li class="usa-nav__primary-item">
+                    <a href="{% url 'admin:user_items' user.get_username %}" class="usa-nav__link">View your items</a>
```

Comment:
Am I right that this only works correctly right now for Django users who happen to be named exactly what appears in the Notes table `created_by` column? I think we could use a comment somewhere in the code reminding us that this link only works in a limited set of circumstances.

Reply:
No, the link will work for any auth'ed user. It just won't return anything unless it can find matches.